core/src/main/java/com/yahoo/oak/Header.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020, Verizon Media.
+ * Licensed under the terms of the Apache 2.0 license.
+ * Please see LICENSE file in the project root for terms.
+ */
+
+package com.yahoo.oak;
+
+import sun.misc.Unsafe;
+
+class Header {
```

Comment:
nit: Shouldn't it be `NativeMemoryManagerHeader`? Or something like that?

Reply:
I do not know yet, as I think Header's design is going to be changed a bit in the future. However, I changed the name to the one that you suggested at least for now.